AK/StackInfo.cpp

```diff
@@ -25,13 +28,23 @@ StackInfo::StackInfo()
         perror("get_stack_bounds");
         VERIFY_NOT_REACHED();
     }
-#elif defined(__linux__)
+#elif defined(__linux__) or defined(__FreeBSD__)
```

Comment:
Will this work on every BSD that's not macOS?

Reply:
I couldn't find a way to query thread attributes on OpenBSD. What other OSes should I check?

Reply:
that was the main one I was thinking of. I'm guessing all the FreeBSD derivatives and NetBSD will have similar APIs, but no users yet :)